coral-common/src/main/java/com/linkedin/coral/common/HiveTable.java

```diff
@@ -186,7 +185,7 @@ private Deserializer getDeserializer() {
   private Deserializer getDeserializerFromMetaStore() {
     try {
       return MetaStoreUtils.getDeserializer(new Configuration(false), hiveTable, false);
-    } catch (MetaException e) {
+    } catch (Throwable e) { // Catch Throwable here because it may throw Exception or Error
```

Comment:
why do we catch and rethrow an RTE here at all when the exception will anyway be caught here https://github.com/linkedin/coral/blob/master/coral-common/src/main/java/com/linkedin/coral/common/HiveTable.java#L166 and logged? can we remove the try catch block?

Reply:
https://github.com/linkedin/coral/blob/master/coral-common/src/main/java/com/linkedin/coral/common/HiveTable.java#L166 also only catches `Exception`, not `Error`.
Compared to throw a Throwable and modify [here](https://github.com/linkedin/coral/blob/master/coral-common/src/main/java/com/linkedin/coral/common/HiveTable.java#L166) to be `Throwable`, I prefer the current approach.